src/common/PropertyExt.js

```diff
@@ -266,6 +266,12 @@
         for (var i in clsArr) {
             if (theme[clsArr[i]]) {
                 for (var paramName in theme[clsArr[i]]) {
+                    if (paramName === "overrideTags" && theme[clsArr[i]][paramName] instanceof Object) {
+                        for (var param in theme[clsArr[i]][paramName]) {
+                            this.tags(param,theme[clsArr[i]][paramName][param]);
```

Comment:
someone was saying we should do it the proper OO way and create a function to make it "less" complex